src/keycloak/chart/values.yaml

```diff
@@ -91,6 +91,24 @@ resources:
     cpu: "1"
     memory: "1Gi"
 
+# Persistence settings for PVC management (when enabled=false, an emptyDir will be used)
+# `devMode` overrides the enabled flag for `data` and `themes` to preserve legacy behavior
```

Comment:
```suggestion
# Persistence settings for PVC management (when enabled=false, an emptyDir will be used)
# `devMode` overrides the enabled flag for `data` and `themes` to allow hot-reloads of the theme/plugin in k3d
```

Also not sure if there's a better way to capture why you might want to toggle this on (either here or in the docs). At first glance I would be afraid someone would see persistence for data and think of course they should turn it on, when really they probably don't need it.

Reply:
I was intentionally a little vague about it, as I'm not sure we necessarily want to encourage what I'm doing 😅 
But it may be worth adding a little more detail regarding the potential need to share/load data from other pods or workloads.

You raise another good point though.. 
When keycloak is running in HA -- do any of these PVCs need to be updated to read-write-many, or is everything hotloaded from the init container?

I'm not sure I'm familiar enough with KC operation to answer that one..

Reply:
All should be loaded from the config image:
- data: import of realm, no additional data gets stored there after
- conf: truststore, always loaded from image
- theme: loaded from image
- provider: plugin jar, also from image

None of them would get used to store additional data so as long as the init image has the above four pieces (it would if built following the indentity-config process) then the emptydir volumes would always get loaded on startup, without the need for a PVC.

Reply:
Your question is basically why I was hoping we could document something to express that most people don't need this 😅. Persistence comes across as something that you would want, but in this case it's not necessary (and potentially could cause probably with multi-replica if it wasn't RWX volumes?).

Reply:
Yea, I think we can add a cautionary note to the README that the default config should be sufficient for MOST deployments and you should really only need to tweak persistence for custom changes/situations.